### DISCUSSION TITLE
Did useBody change to readBody?

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#16998] [Questions]
I was recording a tutorial to show Server API, and  `useBody ` gave me an error that I should use `readBody` instead, so I replaced useBody with readBody on the import from h3, and the error went away and the app worked.

Was useBody renamed to readBody, or is it a bug?

**Edit:**

Some text seems to have been left behind when replacing useBody to readBody? I noticed now...

![image](https://user-images.githubusercontent.com/10191085/187022157-c4372447-ebb1-466a-a17e-49c397ee7a05.png)

**Comment by pi0** [comment]
Hi. Yes it is basically a rename we made. See https://github.com/unjs/h3/pull/156 for more details.
